cluster.yml

```diff
@@ -49,7 +49,7 @@
       setup:
       delegate_to: "{{ item }}"
       delegate_facts: True
-      when: hostvars[item]['ansible_machine_id'] is not defined
+      when: hostvars[item]['ansible_default_ipv4'] is not defined
```

Comment:
Shouldn't this be `ansible_default_ipv4.address`?

Or using something like this:
` when: (hostvars[item] is defined) and (hostvars[item]['ansible_default_ipv4'] is defined) and (hostvars[item]['ansible_default_ipv4']['address'] is defined)`